front/lib/plans/stripe.ts

```diff
@@ -117,7 +120,7 @@ export const createProPlanCheckoutSession = async ({
       workspaceId: owner.id,
     },
   });
-  if (existingSubscription) {
+  if (existingSubscription && !isOldFreePlan(existingSubscription.plan.code)) {
```

Comment:
Looks like the subscription's plan does not exist anymore in prod.

Reply:
If you need the plan information you should include it when fetching the subscription (line 118).